src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -203,4 +203,36 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
         }
       }
     }
+
+  @Path("/{version}/{workflowId}/outputs/{callFqn}")
+  @ApiOperation(
+    value = "Query for call outputs based on workflow id and call fully qualified name (e.g. my_workflow.my_call).",
+    nickname = "call-outputs",
+    httpMethod = "GET",
+    produces = "application/json"
+    //,response = classOf[CallOutputs]
```

Comment:
I assumed this was a reminder to uncomment this once we had a Swagger/Akka HTTP stack that could handle its presence (see @kshakir's findings in the class Scaladoc at the top of this file).